ptx/sec_IBP.ptx

```diff
@@ -1255,7 +1256,7 @@
               $f = Formula("(ln(x+$b))^2")->reduce;
               Context("Fraction");
               Context()->variables->set(x=>{limits=>[-$b,-$b+4]});
-              $F = FormulaUpToConstant("(x+$b)*(ln(x+$b)^2-2*ln(x+$b))")->reduce;
+              $F = FormulaUpToConstant("2*(x+$b)+(x+$b)*(ln(x+$b)^2-2*(x+$b)*ln(x+$b))")->reduce;
```

Comment:
I think this should be:
```suggestion
              $F = FormulaUpToConstant("2x+(x+$b)*ln(x+$b)^2-2*(x+$b)*ln(x+$b)")->reduce;
```

Part of that is just absorbing `2*$b` into the constant of integration. But also the parentheses are such that one term would have `(x+$b)^2` as a factor and I think it should just be `(x+$b)^1`.

Reply:
OK, that works. There is definitely a bracketing error there. The leading `x+$b` is intended to emphasize that this is just a translation of the answer from the earlier example. But I guess WeBWorK won't care if the constant is there or not.